orchagent/srv6orch.cpp

```diff
@@ -130,6 +130,47 @@ bool Srv6Orch::srv6NexthopExists(const NextHopKey &nhKey)
     }
 }
 
+bool Srv6Orch::removeSrv6Nexthops(const std::vector<NextHopGroupKey> &nhgv)
+{
+    SWSS_LOG_ENTER();
+
+    // 1. remove vpn_sid first
+    for (auto& it_nhg : nhgv)
+    {
+        for (auto &sr_nh : it_nhg.getNextHops())
+        {
+            if (sr_nh.isSrv6Vpn())
+            {
+                if (!deleteSrv6Vpn(sr_nh, getAggId(it_nhg)))
+                {
+                    SWSS_LOG_ERROR("Failed to delete SRV6 vpn %s", sr_nh.to_string(false, true).c_str());
```

Comment:
We can also include prefix aggId in the error log for better debugging

Reply:
Fixed, add it in "createSrv6Vpn/deleteSrv6Vpn" error log.